cmd/things/main.go

```diff
@@ -196,6 +208,15 @@ func connectToDB(dbConfig postgres.Config, logger logger.Logger) *sqlx.DB {
 	return db
 }
 
+func createUsersClient(cfg config, logger logger.Logger) (mainflux.UsersServiceClient, func() error) {
+	if cfg.singleUserEmail == "" || cfg.singleUserToken == "" {
```

Comment:
@anovakovic01 this is good, but I would prefer this check:
- If it's sungle user, then return `NewSingleUserService`
- Otherwise proceed as default

That check is logically easier to grasp, because for us defaut would be multi-user, and we do single user only uder some conditions.

Currently your check is a bit different, putting single-suer in the main light, and multi-user only if the single-user checkd do not pass.

Logic is the same, just written differently to make code more understandable.

Reply:
Done.